### PR TITLE
Experimental_Selection_In_Construct

### PR DESCRIPTION
**Pull request by kyx0r**
Shot at fixing https://github.com/FAForever/fa/issues/2288

### CODE REVIEW

File: ext_sector.s [hunk 1]
```diff
@@ -119,8 +119,56 @@ _HOOK_CWldSession__ValidateFocusArmyRequest: ; (int)
...
+	jmp eprLabel
+	
+align 0x8
+_HOOK_ValidateQue:
+	jmp QueLabel	
 ; </ Area for hooks>
 
+align 0x4
+QueLabel:
+	push 0xE19824 ; FACTORY
```
Comment (Eximius) [inline ext_sector.s]: Could be a constant referred by name?

File: ext_sector.s [hunk 2]
```diff
@@ -119,8 +119,56 @@ _HOOK_CWldSession__ValidateFocusArmyRequest: ; (int)
...
+	jmp 0x0128B03D ; jmp to lea ecx, [ss:esp+0x44]
```
Comment (Eximius) [inline ext_sector.s]: Could be a constant referred by name?

File: ext_sector.s [hunk 3]
```diff
@@ -119,8 +119,56 @@ _HOOK_CWldSession__ValidateFocusArmyRequest: ; (int)
...
+	push 0xE204B8 ; EXPERIMENTAL
```
Comment (Eximius) [inline ext_sector.s]: Could be a constant referred by name?

File: ext_sector.s [hunk 4]
```diff
@@ -119,8 +119,56 @@ _HOOK_CWldSession__ValidateFocusArmyRequest: ; (int)
...
+	cmp dx, 2
+	jge 0x006EFB0E
```
Comment (Eximius) [inline ext_sector.s]: Could be a constant referred by name?

**Comment by kyx0r** [comment]
This is deperecated - decided to use a separate category instead of attaching it to experimental.

**Pull request closed without merging**